### DISCUSSION TITLE
Getting error "DNS server not authoritative for zone."

### DISCUSSION THREAD

**Discussion by hodgegw** [Comfy-Org/ComfyUI-Manager#24] [Q&A] [answered]
When trying to install ComfyUI-Manger I get the following error: DNS server not authoritative for zone. (see attached)
![Screenshot 2023-07-11 172453](https://github.com/ltdrdata/ComfyUI-Manager/assets/60791209/f0185377-72bb-4022-8e21-38f3c858cd4b).

I have done all t-shooting for this kind of error on my system, including chkdsk, sfc /scannow. The only thing I have not done as part of a potential fix is to repair/re-install windows. I'm somewhat prepared to do this, but have a doubt as to its viability to resolve this issue.

Does anyone have any idea how I might fix this, please?

Thanks,
Gary

**Comment by ltdrdata** [comment]
https://appuals.com/how-to-resolve-cmd-error-dns-server-not-authoritative-for-zone/

I hope this will solve that issue.

**Reply by hodgegw** [reply, marked as answer]
It did not resolve the issue. 

But, I did some extensive research on the DNS issue, and found that if I install git, it also installs **git-cmd.exe**, which I ran, and then I changed to the directory "custom_nodes", ran the command: git clone https://github.com/ltdrdata/ComfyUI-Manager.git from there and it worked! Wooo Hooo! 
![Screenshot 2023-07-17 170048](https://github.com/ltdrdata/ComfyUI-Manager/assets/60791209/b784debc-8f5a-4e95-a51c-126d5ecfd699)
